Classes/Service/DeeplService.php

```diff
@@ -55,8 +55,12 @@ public function __construct(
         $objectManager = GeneralUtility::makeInstance(ObjectManager::class);
         $this->deeplSettingsRepository = $objectManager->get(SettingsRepository::class);
 
-        $this->loadSupportedLanguages();
-        $this->apiSupportedLanguages['target'] = $this->deeplSettingsRepository->getSupportedLanguages($this->apiSupportedLanguages['target']);
+        $config = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']['wv_deepltranslate'];
```

Comment:
Instead of getting the confVars from $GLOBALS, I'd prefer to use `DeeplBackendUtility`, which has methods set to check for API key, see https://github.com/web-vision/wv_deepltranslate/blob/main/Classes/Utility/DeeplBackendUtility.php#L109

Reply:
@calien666, thanks for the hint! I've changed it ;-)